public/js/wyr-single.js

```diff
@@ -1,14 +1,12 @@
 // Existing code
 const commentBtn = document.querySelector("[name='comment-btn']");
-const closeModal = document.getElementById("close-modal");
+const addCommentCloseModal = document.getElementById("addComment-close-modal");
 const commentModal = document.getElementById("comment-modal");
 const addComment = document.getElementById("add-comment");
 const deleteBtn = document.querySelectorAll('.del');
 
 // Add this line to select the form
-const commentForm = document.querySelector("form");
-
-let commentOpen = false;
+const commentForm = document.getElementById("addCommentForm");
```

Comment:
this is to select the addCommentForm which I updated in the ejs. Although technically this should be selected anyways as when the comment modal is opened, there is only one form on screen, but I wanted to be thorough